pkg/clusterresourceoverride/config.go

```diff
@@ -22,6 +22,9 @@ type ClusterResourceOverrideSpec struct {
 	// LimitRange defaults are merged prior to the override.
 	//
 
+	// EnableSelinuxFix (if true) label pods with svc_t if they have a PVC
+	EnableSelinuxFix bool `json:"enableSelinuxFix"`
```

Comment:
I have mixed feelings regarding the name. `EnableSelinuxFix` may lead to misunderstandings, as not clear what the `fix` is about.
 
Some ideas: `ForceSelinuxRelabel`, `EnsureSelinuxType`, etc.

Reply:
I like ForceSelinuxRelabel ... Changed it to that.